faq.md

```diff
@@ -36,5 +34,9 @@ Use a free service such as [Qualys SSL Labs](https://www.ssllabs.com/ssltest/) t
 
 ## How to reset Nosto module
 
-If the tagging or other Nosto features disappear for example after Prestashop update you need to reset Nosto module. Resetting a module can be done in your store admin. After logging in your store admin navigate to "Modules and service" and locate Nosto module. The location of Nosto module varies a bit depending on Prestashop version. Once you have located the module open the drop down in the module panel and choose "Reset" \(see image below\). Confirm the reset if prompted. After resetting re-connect Nosto account to your store. ![Reset Nosto module](https://user-images.githubusercontent.com/15191701/32780631-b4f75a7e-c94a-11e7-8de3-1761beb959f5.png)
+If the tagging or other Nosto features disappear for example after Prestashop update you need to reset Nosto module. Resetting a module can be done in your store admin. After logging in your store admin navigate to "Modules and service" and locate Nosto module. The location of Nosto module varies a bit depending on Prestashop version. Once you have located the module open the drop down in the module panel and choose "Reset" (see image below). Confirm the reset if prompted. After resetting re-connect Nosto account to your store.
+![Reset Nosto module](https://user-images.githubusercontent.com/15191701/32780631-b4f75a7e-c94a-11e7-8de3-1761beb959f5.png)
+
+## Unable to reconnect Nosto to my shop
 
+If when trying to reconnect existing account you get "Invalid authentication token" or the "Remove Nosto" button does nothing, it is likely that you created multi shop setup after connecting Nosto to the main shop. Resetting the module will resolve the issues.
```

Comment:
```suggestion
If when trying to reconnect an existing account you get "Invalid authentication token" or the "Remove Nosto" button does nothing, it is likely that you created a multi shop setup after connecting Nosto to the main shop. Since prestashop will reset some shop ID's in the database, you'll likely need to reset the Nosto module to resolve the issue.
```